fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/JS/A32NX_Simvars.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * taken from the AAU1 G3000
+ * The purpose is to reduce the amount of string allocations caused by `.toLowerCase()`
+ */
+
+const latlonaltRegEx = new RegExp(/latlonalt/i);
```

Comment:
**thought**: These `RegExp` declarations confuse me. It seems to mix a regular expression literal with a `RegEx` object. If you want a regex which never changes, it's best to use the literal format:

```js
const latlonaltRegEx = /latlonalt/i
```

> Regular expression literals provide compilation of the regular expression when the script is loaded. If the regular expression remains constant, using this can improve performance.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions

Reply:
Done 👍